From: Stephane Glondu <steph@glondu.net>
Date: Wed, 5 Aug 2020 17:21:50 +0200
Subject: Link libtest_functions.so with -lm

Bug: https://github.com/ocamllabs/ocaml-ctypes/issues/655
---
 Makefile.tests | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile.tests b/Makefile.tests
index eb7e45a..2746630 100644
--- a/Makefile.tests
+++ b/Makefile.tests
@@ -1367,7 +1367,7 @@ endif
 
 testlib: $(BUILDDIR)/clib/libtest_functions$(EXTDLL)
 $(BUILDDIR)/clib/libtest_functions$(EXTDLL): $(BUILDDIR)/clib/test_functions.o
-	$(CC) -shared $(LDFLAGS) $(WINLDFLAGS) -o $@ $^
+	$(CC) -shared $(LDFLAGS) $(WINLDFLAGS) -o $@ $^ -lm
 ifneq (,$(filter mingw%,$(OSYSTEM)))
 	cp $@ libtest_functions.dll.a $(BUILDDIR)
 endif
